cloud_formation/lambda/ingest_populate/ingest_queue_upload.py

```diff
@@ -73,21 +79,30 @@ def handler(args, context):
 
         retry = 3
         while retry > 0:
-            resp = queue.send_messages(Entries=batch)
-            sent += len(resp['Successful'])
-
-            if 'Failed' in resp and len(resp['Failed']) > 0:
-                print("Batch failed to enqueue messages")
-                print("Retries left: {}".format(retry))
-                print("Boto3 send_messages response: {}".format(resp))
-                time.sleep(SQS_RETRY_TIMEOUT)
-
-                ids = [f['Id'] for f in resp['Failed']]
-                batch = [b for b in batch if b['Id'] in ids]
+            try:
+                try_again = False
+                resp = queue.send_messages(Entries=batch)
+                if 'Successful' in resp:
```

Comment:
Could the response be something other than `Successful` or `Failed`?

Reply:
I don't think so.  From boto3 docs:
{
    'Successful': [
        {
            'Id': 'string',
            'MessageId': 'string',
            'MD5OfMessageBody': 'string',
            'MD5OfMessageAttributes': 'string',
            'SequenceNumber': 'string'
        },
    ],
    'Failed': [
        {
            'Id': 'string',
            'SenderFault': True|False,
            'Code': 'string',
            'Message': 'string'
        },
    ]
}